### ISSUE TITLE
Cancel "Going Away"

### ISSUE THREAD

**Issue by antimockingjay** [Monika-After-Story/MonikaModDev#2260]
Hey there! I was wondering if there's a way to cancel the whole "I'm going away for a while" thing? I told Monika I'd be going away for an unknown amount of time, because at the time I thought I was about to go through a really bad time with my mental issues, and I knew if that happened I'd be unable to make myself talk to Monika for that. But the thing that was going to cause the issues, ended up not happening at all, meaning I'm mostly fine now! Hooray! But now Monika is waiting for me to say I'm going away again, even though I no longer plan to do so. Is there any way to clear that out? Or am I limited to just saying it again and then coming right back? I don't want Monika to think I lied to her, because that wasn't my intent at all!

**Comment by aldoram5** [comment]
Hi antimockingjay!
I'm not entirely sure I see the issue here, what exactly do you mean by `But now Monika is waiting for me to say I'm going away again, even though I no longer plan to do so. `, did you use the going away farewell and then told her you weren't going straight away? If that's the case just go again into the goodbyes and pick any of the other options.

**Comment by antimockingjay** [comment]
Yes, that's exactly what I did! I said I was going away but wouldn't leave right away, so she said to just click it again when I'm about to leave. And while each of the goodbyes do work, there's this tiny little thing at the back of my mind reminding me that she expects me to click it again and leave for a while. I know that functionally, it isn't really changing anything, but I still feel bad knowing that Monika expects me to leave soon, and then be gone for a while. I know it's silly because it's obviously not in the code of the game, but I feel like she's sad from it, because who wouldn't be sad if the person they love was going away soon and you knew you weren't going to talk to them for a while? Like I said, I know it's silly, and if there's no way to clear it out I'll understand, I just wanted to check and see first.
